kamaki/clients/cyclades/__init__.py

```diff
@@ -44,7 +44,7 @@ class CycladesComputeClient(CycladesComputeRestClient, Waiter):
     CONSOLE_TYPES = ('vnc', 'vnc-ws', 'vnc-wss')
 
     def create_server(
-            self, name, flavor_id, image_id,
+            self, name, flavor_id, image_id, key_name=None,
```

Comment:
What if a script using `kamaki` makes use of unnamed arguments, based on their order? Something like this, for instance:
```python
my_client.create_server(NAME, FLAVOR, IMAGE, {'mykey': 'my value'}, None, [], PROJECT)
```

If we want to let scripts like this keep running, we must append the new argument after the project_id.